start-as-experiment.md

```diff
@@ -0,0 +1,102 @@
+investment.
+
+# Context
+
+The majority of software development projects in the company are distributed
+globally. The company is considering to leverage InnerSource to make the
+required collaboration more efficient. However, most managers are not familiar
+with the Open Source working model and are instead accustomed to hierarchical,
+top-down control style management. The idea of InnerSource is very popular with
+software developers in the company, not the least because many developers use
+or are actively developing Open Source software.
+
+# Forces
+
+- Managers will want to validate the claims of improved collaboration through
+  InnerSource before making a long term investment. This usually involves
+  putting numbers against the expected improvements, which are hard to come
+  by for InnerSource.
+- If the InnerSource initiative will likely have a huge uptake among developers
+  and if many projects are likely to rely on it, a decision to shut it down
+  will be very unpopular and therefore hard to make. The perceived resulting
+  loss of control might deter some managers to even start with InnerSource.
+- Implementing InnerSource style working models are often a radically departure
+  from previously practiced working models. It is therefore likely, that
+  existing, mandatory processes are no longer applicable and that appropriate
+  governing processes are missing. The result might be that one has to operate
+  in a regulatory, sometimes legal no-mans land. Examples are tax and export
+  control related regulations in large corporations with multiple legal
+  entities in multiple countries.
+
+# Solution
+
+Declare the InnerSource initiative as a time limited experiment. Define and
+communicate the criteria for projects to join the InnerSource experiment. The
+criteria should be chosen such that they maximize the chances of building a 
+healthy InnerSource community around the selected InnerSource projects. They
+should also help to ensure that the setting of the projects is such that they 
+can later be used to gain externally valid insights into the effects of 
+applying InnerSource. Examples for such criteria are
+
+- Sufficient geographical distribution of developers
+- Sufficient departmental mix of developers,
+- Openness of communication within community,
+- Career path based on merit within community,
+- Democratic decision making within community.
+
+Consider designating the end of the experiment a _pivot_, _change_ or _pause_
+point to re-evaluate. Also consider establishing a [Review
+Committee](review-committee.md) to increase the chances of management buy-in
+through participation. Depending on company culture, it might be helpful to
+accompany the experiment with appropriate metrics (**tbd: Pattern link**). If
+the projects in the experiment don't provide a direct impact on the companies
+revenue, consider introducing [Cross Team
+Valuation](crossteam-project-valuation.md) to highlight its value contribution.
```

Comment:
Thanks for thinking of this document!  It is not in `master` yet so you'll have to link to [the branch](https://github.com/paypal/InnerSourcePatterns/blob/rrrutledge-patch-3/crossteam-project-valuation.md) for now if you want to reference it.